services/datamanager/datasync/sync.go

```diff
@@ -72,7 +68,7 @@ func NewDefaultManager(logger golog.Logger, cfg *config.Config, lastModMillis in
 			}),
 	}
 
-	conn, err := NewConnection(logger, appAddress, rpcOpts)
+	conn, err := NewConnection(logger, cfg.Cloud.AppAddress, rpcOpts)
```

Comment:
you'll need to check if cloud is in use or not unless NewDefaultManager is only called if cloud management is enabled

Reply:
A little confused here since sync wouldn't work if cloud management is disabled. Where is this enabled/disabled flag?

Reply:
this is may cause a nil ptr exception if you run this locally

Reply:
Ah so should just have a check that `cfg.Cloud != nil` and otherwise create a no-op manager. I'll send out a follow-up. Thanks!